### DISCUSSION TITLE
Generate embeddings functionality?

### DISCUSSION THREAD

**Discussion by bawiz** [EricLBuehler/mistral.rs#671] [General]
Hi Eric,
Will you have some functionality for generating embeddings at some point?
Br, Dara

**Comment by Naveen9182** [comment]
> Hi Eric, Will you have some functionality for generating embeddings at some point? Br, Dara

There are various models that can generate embeddings like BERT, RoBERTa, DistilBERT.

**Comment by EricLBuehler** [comment]
@bawiz @Naveen9182 I've been focusing on other models such as vision (Llama 3.2 vision) or diffusion (FLUX). Embeddings is on the radar!
